### ISSUE TITLE
Selection scope

### ISSUE THREAD

**Issue by Drakone** [pixelhumain/co2#765]
![capture d ecran_2018-03-12_14-10-53](https://user-images.githubusercontent.com/2474636/37277897-7c8b79ba-25ff-11e8-8a79-6979899eb914.png)

J'ai beau essayer de comprendre je ne m'y fait pas du tout. J'ai peur qu'on perde pas mal de gens sur ce coups là donc voilà une proposition.

Pour moi la commune où j'habite n'est qu'un territoire parmis d'autre, je veux simplement l'avoir sous la main si je souhaite visualiser ce qui s'y passe. C'est la définition d'un favoris.
**La ville où j'habite c'est ma ville favorite.**

Voilà ma proposition. Une seule liste, je respire.
![prop2](https://user-images.githubusercontent.com/2474636/37280835-67b820de-2608-11e8-9613-3700f5f84ebf.png)

La commune de communexion est par défaut le premier selectionné.
À la selection d'un lieu il s'ajoute à la liste.

**Comment by RaphaelRIVIERE** [comment]
Yep, on garderais un seul bouttons, si l'utilisateur, change de ville, elle s'ajoute dans mes lieux favories, avec les différents zone géographique associer

Moi j'aime bien l'idée :)

**Comment by Drakone** [comment]
Ouaip, par contre sur l'idée d'ajouter les zone géographique associée je trouve pas ça hyper pertinent (techniquement oui, mais à l'usage je suis pas sûr que ce soit très utile).

Et sur la forme on peut aussi simplement remplacer le logo avant chaque ville par le bouton de suppression comme c'est le cas sur la nouvelle version, je trouve ça hyper intuitif.
![capture d ecran_2018-03-15_13-39-32](https://user-images.githubusercontent.com/2474636/37455489-a0c4f28a-2856-11e8-9fe7-fe7ccceef28a.png)

**Comment by JulienLecaille** [comment]
Ce qui est déroutant c'est la nature de cette localité qui apparaît à côté de "lieux favoris" dont on aperçoit pas nettement la nature: et dont on comprend pas très nettement ce qui se passe quand on clique dessus. Que ce soit juste un favori comme les autres me semble plus clair

**Comment by Drakone** [comment]
Autre truc qui je ne comprend pas. Une fois qu'on a recherché et selectionner un scope, comment revenir à l'état par défaut (aucun scope selectionné) ?

La seule solution que j'ai trouvé est de cliquer deux fois sur _Mes lieux favoris_. Mais c'est pas du tout intuitif...
![peek 17-04-2018 09-38](https://user-images.githubusercontent.com/2474636/38850757-e360e6bc-4223-11e8-9518-f659f3a896ef.gif)

**Comment by RaphaelRIVIERE** [comment]
Je comprends pas par l'état par défaut 

Pour n'avoir qu'un bouttons et pour quoi réavoir mes tags favoris

**Comment by Drakone** [comment]
État par défaut = état lorsqu'on arrive sur le site = aucun lieu selectionné.

**Comment by Drakone** [comment]
@clement59 @Kgneo @oceatoon @JulienLecaille @SimonSarazin @MaiaDereva @Jahne-Anders 

Qu'en pensez-vous ? Comment utilisez-vous cet outil ?

**Comment by MaiaDereva** [comment]
J'avoue que l'aspect territorial n'est pas un critère pour moi qui suis relativement nomade... donc je ne me sers pas de cet outil. Mais je n'ai pas le profil d'un utilisateur lambda...

**Comment by Drakone** [comment]
Design de la nouvelle version : 

![capture d ecran_2018-11-28_09-44-23](https://user-images.githubusercontent.com/2474636/49139514-4a752a00-f2f2-11e8-8206-a85535ffce59.png)

Ça manque un peu de marges je pense.
Et surtout un bouton pour désélectionner tous les scope.
